sdk/php/dev/dagger.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "php-sdk-dev",
-  "engineVersion": "v0.19.3",
+  "engineVersion": "v0.19.4",
   "sdk": {
     "source": "php"
-  }
+  },
+  "disableDefaultFunctionCaching": true
```

Comment:
Missed a spot - or is this because the PHP SDK doesn't support the pragmas yet?

Reply:
nope just missed it, will fix!